runtime/example_init.lua

```diff
@@ -0,0 +1,85 @@
+-- Set <space> as the leader key See `:help mapleader` NOTE: Must happen before plugins are loaded
+-- (otherwise wrong leader will be used)
+vim.g.mapleader = ' '
+
+-- [[ Setting options ]] See `:h vim.o`
+-- NOTE: You can change these options as you wish!
+-- For more options, you can see `:help option-list`
+-- To see documentation for an option, you can use `:h 'optionname'`, for example `:h 'number'`
+-- (Note the single quotes)
+
+-- Print the line number in front of each line
+vim.o.number = true
+
+-- Use relative line numbers, so that it is easier to jump with j, k. This will affect the 'number'
+-- option above, see `:h number_relativenumber`
+vim.o.relativenumber = true
+
+-- Sync clipboard between OS and Neovim. Schedule the setting after `UiEnter` because it can
+-- increase startup-time. (see `:h vim.schedule()`). Remove this option if you want your OS
+-- clipboard to remain independent. See `:help 'clipboard'`
+vim.schedule(function()
```

Comment:
this performance note seems kind of "in the weeds", but if we're going to keep it, do we want to use an actual UIEnter handler instead of this accidental feature of `vim.schedule()` (which introduces bad/confusing habits, the opposite of what we want to do in this sample config)